projects/core/directives/dropdown/dropdown-context.directive.ts

```diff
@@ -34,9 +34,9 @@ const MOVE_THRESHOLD = 15;
             'onTouchMove($event.touches[0].clientX, $event.touches[0].clientY)',
         '(touchstart.silent.passive)':
             'onTouchStart($event.touches[0].clientX, $event.touches[0].clientY)',
-        '(document:pointerdown.silent)': 'closeDropdown($event.target)',
-        '(document:contextmenu.capture.silent)': 'closeDropdown($event.target)',
-        '(document:keydown.esc)': 'closeDropdown($event.target)',
+        '(document:pointerdown.silent)': 'closeDropdown($event.currentTarget)',
```

Comment:
It was:
```ts
    @HostListener('document:pointerdown.silent', ['$event.target'])
    @HostListener('document:contextmenu.capture.silent', ['$event.target'])
    @HostListener('document:keydown.esc', ['$event.currentTarget'])
```
So `currentTarget` only on the last one.

Reply:
fixed